realm/realm-library/src/main/cpp/object-store/src/schema.hpp

```diff
@@ -51,6 +51,8 @@ class Schema : private std::vector<ObjectSchema> {
     using base::end;
     using base::empty;
     using base::size;
+    using base::push_back;
```

Comment:
`push_back()` in particular is not a sensible thing to expose directly as the ObjectSchema are stored in sorted order.